src/Tasks/ResolveKeySource.cs

```diff
@@ -158,7 +158,7 @@ private bool ResolveAssemblyKey()
                             fs?.Close();
                         }
 #else
-                        Log.LogError("PFX signing not supported on .NET Core");
+                        Log.LogErrorWithCodeFromResources("ResolveKeySource.PfxUnsupported");
```

Comment:
Strictly speaking assembly signing can take a snk file or a pfx file. So the error message about PfxUnsupported in not totally accurate. "Assembly Signing not supported on .NET Core" might be more accurate.

Reply:
This code path is specific for `.pfx` (condition on line 108) - so while I'm not strongly opinionated about it, it doesn't feel bad to me to be specific in the error message

Reply:
[Resolved offline] specifying pfx in error message here is desired.